src/annotator/test/guest-test.js

```diff
@@ -313,71 +285,28 @@ describe('Guest', () => {
     });
   });
 
-  describe('events from sidebar frame', () => {
```

Comment:
We had an existing set of tests for handling "scrollToAnnotation" requests from the sidebar. Here I have extracted those tests out of the "events from sidebar frame" block into a separate block which re-uses the same tests for when the event comes from the sidebar and when it comes from the host.